pyccl/tests/test_tkkssc.py

```diff
@@ -249,3 +249,146 @@ def test_tkkssc_counterterms_gc(kwargs):
 
     assert np.abs((tk_nogc_12 - tkc12) / tk_gc_12 - 1).max() < 1e-5
     assert np.abs((tk_nogc_34 - tkc34) / tk_gc_34 - 1).max() < 1e-5
+
+
+@pytest.mark.parametrize('kwargs', [{'is_number_counts1': True,
+                                     'is_number_counts2': True,
+                                     'is_number_counts3': True,
+                                     'is_number_counts4': True},
+                                    {'is_number_counts1': True,
+                                     'is_number_counts2': True,
+                                     'is_number_counts3': True,
+                                     'is_number_counts4': False},
+                                    {'is_number_counts1': True,
+                                     'is_number_counts2': True,
+                                     'is_number_counts3': False,
+                                     'is_number_counts4': False},
+                                    {'is_number_counts1': True,
+                                     'is_number_counts2': True,
+                                     'is_number_counts3': False,
+                                     'is_number_counts4': True},
+                                    {'is_number_counts1': True,
+                                     'is_number_counts2': False,
+                                     'is_number_counts3': False,
+                                     'is_number_counts4': True},
+                                    {'is_number_counts1': True,
+                                     'is_number_counts2': False,
+                                     'is_number_counts3': False,
+                                     'is_number_counts4': False},
+                                    {'is_number_counts1': True,
+                                     'is_number_counts2': False,
+                                     'is_number_counts3': True,
+                                     'is_number_counts4': False},
+                                    {'is_number_counts1': False,
+                                     'is_number_counts2': False,
+                                     'is_number_counts3': True,
+                                     'is_number_counts4': False},
+                                    ])
+def test_tkkssc_linear_bias(kwargs):
+    hmc = ccl.halos.HMCalculator(COSMO, HMF, HBF, mass_def=M200,
+                                 nlog10M=2)
+    k_arr = KK
+    a_arr = np.array([0.3, 0.5, 0.7, 1.0])
```

Comment:
if this test takes too long it'd be fine to do it for a single redshift

Reply:
It's pretty fast. I'll leave it as is.